astroplan/tests/test_core.py

```diff
@@ -786,6 +741,31 @@ def test_TargetNeverUpWarning(recwarn):
     assert issubclass(w.category, TargetNeverUpWarning)
     assert np.isnan(no_time)
 
+def test_get_site():
+    # Compare to the IRAF observatory list available at:
+    # http://tdc-www.harvard.edu/iraf/rvsao/bcvcorr/obsdb.html
+    keck = get_site('keck')
+    lon, lat, el = keck.to_geodetic()
+    assert_allclose(lon, -1*Longitude('155:28.7', unit=u.deg).value, atol=0.001)
+    assert_allclose(lat, Latitude('19:49.7', unit=u.deg).to(u.deg).value,
+                    atol=0.001)
+    assert_allclose(el, 4160, atol=1)
+
+    keck = get_site('ctio')
+    lon, lat, el = keck.to_geodetic()
+    assert_allclose(lon, -1*Longitude('70.815', unit=u.deg).value, atol=0.001)
```

Comment:
Is `lon` a `Longitude` or `Angle` object?
If yes, I'd be +1 to call [assert_quantity_allclose](http://astropy.readthedocs.org/en/latest/api/astropy.tests.helper.assert_quantity_allclose.html) here (and in similar cases where you do asserts on quantities.